rasa/cli/x.py

```diff
@@ -110,46 +110,64 @@ def _prepare_credentials_for_rasa_x(
 def _overwrite_endpoints_for_local_x(
     endpoints: AvailableEndpoints, rasa_x_token: Text, rasa_x_url: Text
 ):
-    from rasa.utils.endpoints import EndpointConfig
-    import questionary
+    endpoints.model = _get_model_endpoint(endpoints.model, rasa_x_token, rasa_x_url)
+    endpoints.event_broker = _get_event_broker_endpoint(endpoints.event_broker)
 
-    # Checking if endpoint.yml has existing url and wait time values set, if so give
-    # warning we are overwriting the endpoint.yml file.
-    custom_wait_time_pulls = endpoints.model.kwargs.get("wait_time_between_pulls")
-    custom_url = endpoints.model.url
+
+def _get_model_endpoint(
+    model_endpoint: Optional[EndpointConfig], rasa_x_token: Text, rasa_x_url: Text
+) -> EndpointConfig:
     default_rasax_model_server_url = (
         f"{rasa_x_url}/projects/default/models/tag/production"
     )
 
-    if custom_url != default_rasax_model_server_url:
-        warnings.warn(
+    model_endpoint = model_endpoint or EndpointConfig()
+
+    # Checking if endpoint.yml has existing url, if so give
+    # warning we are overwriting the endpoint.yml file.
+    custom_url = model_endpoint.url
+
+    if custom_url and custom_url != default_rasax_model_server_url:
+        logger.info(
             f"Ignoring url '{custom_url}' from 'endpoints.yml' and using "
             f"'{default_rasax_model_server_url}' instead."
         )
```

Comment:
It's not really part of `things a user should fix on his side`, nor `things that are not optimal, but where the user can not immediately fix them` (warnings.warn and logger.warning), I propose this just being info

Reply:
I mean they can just leave it empty or put the correct URL there, right?

Reply:
However, info is also fine to me

Reply:
What they have there isn't necessarily wrong though, right? If it's the url they use when they run rasa open source, but it doesn't work for rasa x, they shouldn't have to change it?

Reply:
haha it did not show the second. if info is fine, feel free to merge when green

Reply:
Yeah, it's a 50:50 thing

Reply:
or change it back 😃only changed the code directly because you told me to when you left, it felt weird haha. If you don't agree we can also switch back

Reply:
I just am wary of `warnings.warn` warnings that don't tell the user how to fix the issue